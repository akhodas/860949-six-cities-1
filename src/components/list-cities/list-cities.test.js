import React from 'react';
import renderer from 'react-test-renderer';

import ListCities from './list-cities.jsx';


describe(`ListCities`, () => {
  it(`correctly rendered`, () => {
    const tree = renderer
    .create(<ListCities
      selectedCity = {`Minsk`}
      listCities = {[`Minsk`, `London`, `Paris`]}
      onCityClick = {jest.fn()}
    />)
    .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
