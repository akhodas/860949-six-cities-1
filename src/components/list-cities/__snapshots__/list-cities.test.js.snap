// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListCities correctly rendered 1`] = `
<section
  className="locations container"
>
  <ul
    className="locations__list tabs__list"
  >
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item tabs__item--active"
        href="#"
        onClick={[Function]}
      >
        <span>
          Minsk
        </span>
      </a>
    </li>
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item "
        href="#"
        onClick={[Function]}
      >
        <span>
          London
        </span>
      </a>
    </li>
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item "
        href="#"
        onClick={[Function]}
      >
        <span>
          Paris
        </span>
      </a>
    </li>
  </ul>
</section>
`;
